src/celeritas/global/CoreTrackData.cc

```diff
@@ -35,6 +35,9 @@ void resize(CoreStateData<Ownership::value, M>* state,
     resize(&state->sim, size);
     resize(&state->init, params.init, size);
     resize(&state->track_slots, size);
+    // TODO: Get number of actions + 1 from CoreParams
+    resize(&state->thread_offsets, 32);
```

Comment:
As discussed this morning, we should somehow get the # of actions.

Reply:
See #769 .